packages/shared/src/components/modals/ShareModal.spec.tsx

```diff
@@ -23,76 +38,141 @@ Object.assign(navigator, {
 beforeEach(async () => {
   nock.cleanAll();
   jest.clearAllMocks();
+  features = defaultFeatures;
 });
 
-const renderComponent = (comment?): RenderResult => {
+const squads = [generateTestSquad()];
+
+const renderComponent = (
+  loggedIn = true,
+  hasSquads = true,
+  comment?,
+): RenderResult => {
   const client = new QueryClient();
 
   return render(
     <QueryClientProvider client={client}>
-      <AuthContextProvider user={null} squads={[]}>
-        <ShareModal
-          origin={Origin.Feed}
-          post={defaultPost}
-          comment={comment}
-          isOpen
-          onRequestClose={onRequestClose}
-          ariaHideApp={false}
-        />
-      </AuthContextProvider>
+      <FeaturesContextProvider flags={features}>
+        <AuthContextProvider
+          user={loggedIn ? loggedUser : null}
+          updateUser={jest.fn()}
+          tokenRefreshed
+          getRedirectUri={jest.fn()}
+          loadingUser={false}
+          loadedUserFromCache
+          squads={hasSquads ? squads : []}
+        >
+          <LazyModalElement />
+          <ShareModal
+            origin={Origin.Feed}
+            post={defaultPost}
+            comment={comment}
+            isOpen
+            onRequestClose={onRequestClose}
+            ariaHideApp={false}
+          />
+        </AuthContextProvider>
+      </FeaturesContextProvider>
     </QueryClientProvider>,
   );
 };
 
-it('should render the article card', async () => {
-  renderComponent();
-  expect(screen.getByAltText(`${defaultPost.title}`)).toBeInTheDocument();
-  expect(
-    screen.getByAltText(`Avatar of ${defaultPost.source.handle}`),
-  ).toBeInTheDocument();
-  expect(screen.getByText(defaultPost.title)).toBeInTheDocument();
-});
+describe('ShareModal Test Suite:', () => {
```

Comment:
Just a future note, this is slightly redundant as the file will act as a suite already.
Normally we only use describe when it's multi purpose tests inside a file.

But you can leave it for now.

Reply:
Thank you, I'll follow this going forward. Iv'e always used the describe to group the tests even if we are only testing one thing in the file